apps/api/src/domain/models/audio-item/commands/transcripts/import-line-items-to-transcript/import-line-items-to-transcript.command.ts

```diff
@@ -0,0 +1,66 @@
+import {
+    ICommandBase,
+    LanguageCode,
+    ResourceCompositeIdentifier,
+    ResourceType,
+} from '@coscrad/api-interfaces';
+import { Command } from '@coscrad/commands';
+import { NestedDataType, NonEmptyString, NonNegativeFiniteNumber } from '@coscrad/data-types';
+import { LanguageCodeEnum } from '../../../../../../domain/common/entities/multilingual-text';
+import { AudioVisualCompositeIdentifier } from '../../../entities/audio-item-composite-identifier';
+import { IMPORT_LINE_ITEMS_TO_TRANSCRIPT } from '../constants';
+
+class TranscriptLineItem {
```

Comment:
Might be nice if there were a way to share this with the `AddLineItemToTranscript` command. But let's not worry about that until everything is done and working.